shared/devices/device-revoke/index.js

```diff
@@ -71,33 +100,56 @@ const DeviceRevoke = (props: Props) => (
   />
 )
 
-const styleHeader = {
-  marginBottom: globalMargins.tiny,
-}
-
-const styleTLF = {
-  marginBottom: globalMargins.xtiny,
-}
-
-const styleName = {
-  color: globalColors.red,
-  fontStyle: 'italic',
-  marginTop: 4,
-  textDecorationLine: 'line-through',
-}
-
-const styleDevicesContainer = {
-  ...globalStyles.flexBoxColumn,
-  alignItems: 'flex-start',
-  alignSelf: 'center',
-  border: '1px solid ' + globalColors.black_05,
-  borderRadius: 4,
-  height: 162,
-  marginBottom: globalMargins.small,
-  marginTop: globalMargins.small,
-  overflowY: 'scroll',
-  padding: globalMargins.small,
-  width: 440,
-}
+const styles = styleSheetCreate({
+  bodyText: {
+    textAlign: 'center',
+  },
+  bodyTextContainer: {
+    marginBottom: globalMargins.tiny,
+  },
+  header: {
+    ...globalStyles.flexBoxColumn,
+    alignItems: 'center',
+  },
+  headerDeviceName: {
+    color: globalColors.red,
+    fontStyle: 'italic',
+    marginTop: 4,
+    textDecorationLine: 'line-through',
+  },
+  tlfEntry: platformStyles({
+    common: {
+      flexDirection: 'row',
+      marginBottom: globalMargins.xtiny,
+    },
+    isElectron: {
+      textAlign: 'left',
+    },
+  }),
+  tlfList: {
+    margin: globalMargins.small,
+    flex: 1,
+  },
+  tlfListContainer: platformStyles({
+    common: {
+      ...globalStyles.flexBoxColumn,
+      alignContent: 'center',
+      borderColor: globalColors.black_05,
+      borderRadius: 4,
+      borderStyle: 'solid',
+      borderWidth: 1,
+      marginBottom: globalMargins.small,
+      marginTop: globalMargins.small,
+    },
+    isElectron: {
+      height: 162,
+      width: 440,
+    },
+    isMobile: {
+      height: 176,
```

Comment:
The endangeredTLFs list is currently kinda buggy and I think height was set sort of implicitly before, so I decided to just set a list height of 176, which looks pretty good on all phone sizes.

Reply:
can you show me this